docs/antora/build/site/_/img/Chronicle-Logo-White.svg

```diff
@@ -0,0 +1 @@
.3-1.5-3.9c0-1.5.5-2.8 1.6-3.9s2.3-1.6 3.8-1.6 2.8.5 3.8 1.6c1.1 1.1 1.6 2.3 1.6 3.9-.1 1.5-.6 2.8-1.6 3.9zm-7.9 57.5V55.4h8v48.2h-8z" class="st0"/><path id="path4289" d="M264.8 97.6c-4.3-4.6-6.5-10.6-6.5-18.1 0-7.5 2.2-13.5 6.5-18 4.3-4.6 10-6.9 16.9-6.9 5.8 0 10.5 1.3 14.2 4 3.7 2.7 6.2 6.5 7.7 11.5H295c-.9-2.8-2.5-5-4.8-6.6-2.3-1.6-5.1-2.4-8.4-2.4-4.6 0-8.2 1.6-11 4.8-2.8 3.2-4.2 7.7-4.2 13.5s1.4 10.3 4.2 13.5c2.8 3.2 6.5 4.8 11 4.8 6.9 0 11.3-3 13.3-9h8.6c-1.5 4.7-4.1 8.5-7.8 11.3-3.7 2.8-8.4 4.2-14.1 4.2-7 .2-12.6-2.1-17-6.6z" class="st0"/><path id="path4291" d="M308.9 103.6v-65h8v65h-8z" class="st0"/><path id="path4293" d="M345 104.4c-6.9 0-12.5-2.3-16.9-6.9-4.3-4.6-6.5-10.6-6.5-18.1 0-7.5 2.2-13.5 6.6-18 4.4-4.5 10.1-6.8 17.1-6.8 7 0 12.6 2.2 16.9 6.5 4.2 4.3 6.4 9.8 6.4 16.5 0 1.7-.1 3.3-.4 4.7h-38.4c.2 4.8 1.7 8.6 4.5 11.4 2.8 2.8 6.3 4.2 10.7 4.2 3.3 0 6-.7 8.3-2.2 2.3-1.5 3.9-3.4 5-5.8h8.6c-1.5 4.2-4.1 7.7-7.9 10.5-3.7 2.6-8.3 4-14 4zm10.8-39c-2.9-2.8-6.4-4.2-10.7-4.2-4.2 0-7.8 1.4-10.6 4.1-2.8 2.7-4.4 6.5-4.7 11.2h30.3c.1-4.7-1.4-8.4-4.3-11.1z" class="st0"/></g></g><g id="g4317"><g id="g4315"><path id="path4299" d="M201.2 122.7c3.8.9 5.7 2.3 5.7 5.3 0 3.4-2.6 5.4-6.4 5.4-2.7 0-5.3-.9-7.4-2.9l1.9-2.2c1.7 1.5 3.4 2.3 5.6 2.3 1.9 0 3.2-.9 3.2-2.3 0-1.3-.7-2-4-2.8-3.8-.9-5.9-2-5.9-5.4 0-3.1 2.5-5.2 6.1-5.2 2.6 0 4.7.8 6.5 2.2l-1.7 2.4c-1.6-1.2-3.2-1.8-4.8-1.8-1.8 0-2.9.9-2.9 2.1-.2 1.5.7 2.1 4.1 2.9z" class="st0"/><path id="path4301" d="M221.4 133.4c-5.5 0-9.3-4.2-9.3-9.2 0-5 3.9-9.3 9.4-9.3s9.3 4.2 9.3 9.2c0 5.1-3.9 9.3-9.4 9.3zm0-15.6c-3.5 0-6 2.8-6 6.3s2.5 6.4 6.1 6.4 6-2.8 6-6.3-2.5-6.4-6.1-6.4z" class="st0"/><path id="path4303" d="M250.2 118.1H240v4.9h9v2.9h-9v7.3h-3.1v-17.8h13.3v2.7z" class="st0"/><path id="path4305" d="M264 133.1h-3.2v-14.9h-5.7v-2.9h14.5v2.9H264v14.9z" class="st0"/><path id="path4307" d="M285.9 115.2h2.7l4.4 13.3 4.3-13.2h3.3l-6.2 18h-2.7l-4.4-12.8-4.4 12.8h-2.7l-6.2-18h3.4l4.3 13.2 4.2-13.3z" class="st0"/><path id="path4309" d="M320.3 133.1H317l-1.8-4.3h-8.4l-1.8 4.3h-3.2l7.9-18h2.9l7.7 18zm-9.4-14.2l-3.1 7.1h6.1l-3-7.1z" class="st0"/><path id="path4311" d="M337.1 133.1l-4.4-6.2h-3.9v6.2h-3.1v-17.8h8c4.1 0 6.6 2.2 6.6 5.7 0 3-1.8 4.7-4.3 5.4l4.8 6.8h-3.7zm-3.8-15h-4.6v6h4.6c2.2 0 3.7-1.2 3.7-3 0-2-1.4-3-3.7-3z" class="st0"/><path id="path4313" d="M359.6 118.1h-10.1v4.6h8.9v2.8h-8.9v4.8h10.2v2.8h-13.4v-17.8h13.2v2.8z" class="st0"/></g></g></g><g id="g4329"><path id="path4321" d="M498.8 17.9l-5.8-12c0-.6-.6-1.8-1.4-2.2-.8-.4-1.9-.6-3.4.4l-17 12.3c-1.6 1.1-1.9 1.6-2.9 3.3L443.9 73l16.4 29.9c-.1.1.2-.3.5-.7l37.9-78.8c.7-1.6.9-3.1.1-5.5z" class="st0"/><path id="path4323" d="M431.9 57.8l-.7-1.4-.2-.3c-.1-.2-.3-.4-.4-.6-.7-.8-1.6-1.2-2.5-1.2H406c-1.3 0-2.5.8-3.1 2.2l-33.7 71.1c-.6 1.2-.6 2.7 0 3.9.6 1.2 1.7 2 2.9 2.1h21.4c1.8 0 3.4-.7 4.5-2.4l19.8-42.8 13.8-29.8.3-.8z" class="st0"/><path id="path4325" d="M457.4 104.3l-23.5-43.7-.2.4-13.9 30 5 10.2c1.4 2.3 3.6 3 5.2 3h4.8l22.6.1z" class="st0"/></g></g></g></switch></svg>
```

Comment:
we shouldn't commit the generated site - this is gonna be posted elsewhere yes?

Reply:
We should ignore this dir

Reply:
Will remove and ignore in my next commit!